src/state.ts

```diff
@@ -76,13 +102,6 @@ const state = new Proxy(overlay, {
 
         // Persist "sets" to storage in the background for some keys
         if (PERSISTENT_KEYS.includes(property)) {
-            // Ensure we don't accidentally store anything sensitive
-            if (browser.extension.inIncognitoContext) {
-                console.error(
-                    "Attempted to write to storage in private window.",
-                )
-                return false
-            }
```

Comment:
I'm pretty nervous about this change - I need to look at it really carefully and make sure it is safe :)